Functions/PollEmailProcessing.cs

```diff
@@ -0,0 +1,60 @@
+using System.Text.RegularExpressions;
+using Microsoft.Azure.Functions.Worker;
+using Microsoft.Extensions.Logging;
+
+namespace Functions
+{
+    public class PollEmailProcessing
+    {
+        private readonly ILogger _logger;
+
+        public PollEmailProcessing(ILoggerFactory loggerFactory)
+        {
+            _logger = loggerFactory.CreateLogger<PollEmailProcessing>();
+        }
+
+        [Function("PollEmailProcessing")]
+        public async Task RunAysnc([TimerTrigger("* * * * *")] MyInfo myTimer)
+        {
+            var url = Environment.GetEnvironmentVariable("PollEmailProcessingUrl", EnvironmentVariableTarget.Process);
+            url ??= "https://find-a-tuition-partner-qa.london.cloudapps.digital/admin/process-emails";
+
+            _logger.LogInformation($"Find a Tuition Partner PollEmailProcessing function executed at: {DateTime.UtcNow}.  Polling {url}");
+
+            var httpClient = new HttpClient(new HttpClientHandler() { UseDefaultCredentials = true });
```

Comment:
It's recommended to use `IHttpClientFactory` for creating `HttpClient` instances in a managed, resilient way. This helps to prevent socket exhaustion that can occur when directly instantiating `HttpClient`.

Here is an example,

        ```
        private readonly ILogger _logger;
        private readonly HttpClient _httpClient;

        public PollEmailProcessing(ILoggerFactory loggerFactory, IHttpClientFactory httpClientFactory)
        {
            _logger = loggerFactory.CreateLogger<PollEmailProcessing>();
            _httpClient = httpClientFactory.CreateClient();
        }
```

Reply:
Added DI